cmd/internal/cmd/flow/flow.go

```diff
@@ -101,7 +103,22 @@ type node struct {
 }
 
 func runFlows(p *printer.Printer) error {
-	ts := time.Now().UTC().Add(-1 * opts.since)
+	now := time.Now().UTC()
+	until := now.Add(-opts.until)
+	since := now.Add(-opts.since)
+
+	if opts.count < 1 {
+		return errors.New("--count must be at least 1")
+	}
+	if since.After(until) {
```

Comment:
nitpicking: I slightly prefer `!until.After(since)` here as it will take the branch when `since == until`

Reply:
Ah good point